sail/blueprints/sample.yaml

```diff
@@ -0,0 +1,71 @@
+---
+version: 1
+
+# Install plugins from the WordPress.org repo or custom sources.
+plugins:
+  woocommerce: latest
```

Comment:
- [feat] plugin auto activation option

Reply:
@DimaMinka by default all plugins specified in a blueprint will be activated after install. Is there a good use case where that would not be ideal?